src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -9117,3 +9117,55 @@ def test_aks_maintenanceconfiguration(self, resource_group, resource_group_locat
         # delete
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
+
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_update_upgrade_settings(self, resource_group, resource_group_location):
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=78661070&view=results) to validate the change.

Reply:
Requeued the [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=79708965&view=results)

Reply:
test passed